src/prefect/environments/execution/remote/environment.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any
+
+import cloudpickle
+
+from prefect.environments.execution import Environment
+from prefect.environments.storage import Docker
+from prefect.utilities import logging
+from prefect.utilities.configuration import set_temporary_config
+
+
+class RemoteEnvironment(Environment):
+    """
+    RemoteEnvironment is an environment which takes in information around an executor
+    and runs the flow in place using that executor.
+
+    Example using a RemoteEnvironment w/ an existing Dask cluster:
+    ```
+    RemoteEnvironment(
+        executor="prefect.engine.executors.DaskExecutor",
+        executor_kwargs={"address": "tcp://dask_scheduler_address"}
+    )
+    ```
+
+    Args:
+        - executor (str, optional): an importable string to an executor class; defaults
+            to `prefect.engine.executors.SynchronousExecutor`
+        - executor_kwargs (dict, optional): a dictionary of kwargs to be passed to
+            the executor; defaults to an empty dictionary
+    """
+
+    def __init__(
+        self,
+        executor: str = "prefect.engine.executors.SynchronousExecutor",
+        executor_kwargs: dict = dict(),
```

Comment:
Because dictionaries are mutable, you should default it to `None` here to avoid weird statefulness, and then set it as a `dict()` below using:
```python
self.executor_kwargs = executor_kwargs or dict()
```